addons/account/models/account_move.py

```diff
@@ -117,6 +117,21 @@ def _sequence_fixed_regex(self):
         tracking=True,
         default='draft',
     )
+
+    visualization_state = fields.Selection(
+        selection=[
+            ('draft', 'Draft'),
+            ('posted', 'Posted'),
+            ('sent', 'Sent'),
+            ('cancel', 'Cancelled'),
+        ],
+        required=True,
+        readonly=True,
+        copy=False,
+        tracking=True,
```

Comment:
We already track `is_move_sent` and `state`, so I feel this is a bit redundant
![image](https://github.com/odoo/odoo/assets/32981518/5db33e25-3203-4620-b009-89ed1b767bba)